dbt_subprojects/dex/macros/models/_project/uniswap_compatible_trades_no_factory.sql

```diff
@@ -0,0 +1,77 @@
+{% macro uniswap_compatible_v2_trades_no_factory(
```

Comment:
We may retry with this toy macro after transfers updated.

Reply:
i don't know, think we should try to stick to simple fork. as noted above, let me check in with the team.